src/sync/incomingSyncUtils.js

```diff
@@ -193,6 +193,14 @@ export const sanityCheckIncomingRecord = (recordType, record) => {
       cannotBeBlank: ['name', 'number_to_use'],
       canBeBlank: [],
     },
+    Pref: {
+      cannotBeBlank: ['ID', 'store_ID', 'item', 'data'],
+      canBeBlank: ['user_ID', 'network_ID'],
+    },
+    PrefBlob: {
+      cannotBeBlank: ['ID', 'store_id', 'item', 'type', 'blob'],
+      canBeBlank: ['user_ID', 'network_id'],
```

Comment:
we don't integrated pref blob records, was wondering why we would need to do sanity check on them

Reply:
Fair point. 615d1b2a42f7545707699d21d693733763b1c32e.